lib-multisrc/greenshit/src/eu/kanade/tachiyomi/multisrc/greenshit/GreenShit.kt

```diff
@@ -105,11 +107,18 @@ abstract class GreenShit(
         val url = "$apiUrl/obras/novos-capitulos".toHttpUrl().newBuilder()
             .addQueryParameter("pagina", page.toString())
             .addQueryParameter("limite", "24")
-            .addQueryParameter("gen_id", "4")
+            .addQueryParameterIf(targetAudience != TargetAudience.All, "gen_id", targetAudience.toString())
             .build()
         return GET(url, headers)
     }
 
+    private fun HttpUrl.Builder.addQueryParameterIf(condition: Boolean, name: String, value: String): HttpUrl.Builder {
```

Comment:
Is this argument order a well-known convention? Otherwise I would say the condition / predicate should be at the end. Especially when you read the function name plainly, 'Add query parameter {name}:{value} if {condition}'.